roles/tendrl-repo/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Could we refer to the development builds as develop and not devel? Something like `s/devel/develop/` in this patch would be required. I would rather use the exact same name as used for both copr builds and branch.